### ISSUE TITLE
iOS Chip Accessibility Issues

### ISSUE THREAD

**Issue by kotavy** [callstack/react-native-paper#2056]
### Current behaviour

The Chip component is not accessible as expected. `accessibilityLabel` reads out, but `accessibilityHint` does not. `onClose` cannot be accessed by iOS VoiceOver.

This may be an issue for Android as well, but this project is built solely for iOS so I can't say.

### Expected behaviour

The `accessibilityHint` should be read out in VoiceOver. `onClose` should be accessible, either by enabling selection of both the chip content and the close button, or by propagating taps to the close button.

### Code sample

      <Chip
        accessibilityHint='Hints are currently not read.'
        accessibilityLabel='Labels are read.'
        // onPress offers a workaround, but some users may want both onPress and onClose
        onPress={() => onClose()}
        onClose={() => onClose()}
        mode='outlined'
      >
        My Chip
      </Chip>

### What have you tried

Inspecting the Chip structure, there doesn't seem to be a way to change the accessibility settings of the components that build the Chip itself. `onAccessibilityTap` did not work either.

I would like to update my versions, but this is not an independent project and I would have to show evidence that the issue would be resolved in order to justify the updates.

I searched the repo but could not find a similar issue, either open or closed. A web search also turned up sparse results for Chip accessibility.

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | iOS
| react-native          | 0.60.5
| react-native-paper    | 3.5.1
| react-native-vector-icons    | 6.6.0
| node                  | 12.14.1
| npm or yarn           | yarn 1.17.3
| expo sdk              | N/A

**Comment by brunohkbx** [comment]
Hey @kotavy. This should be fixed in the next release :smiley:
